src/lucky/remote_ip_address.cr

```diff
@@ -0,0 +1,15 @@
+module Lucky::RemoteIpAddress
+  # Returns a `String` IP Address of the remote client by
+  # looking at a few different possible options.
+  #
+  # * HTTP_X_FORWARDED_FOR header
+  # * REMOTE_ADDRESS header
+  # * Fallback to localhost 127.0.0.1
+  def remote_ip : String
+    request = @context.request
+
+    request.headers["HTTP_X_FORWARDED_FOR"]?.try(&.split(',').first) ||
```

Comment:
There is a potential bug here. If HTTP_X_FORWARDED_FOR is an empty string it will try to split it and will return an empty string. See `"".try(&.split(',').first) # => ""`.

I think if you added a spec for an empty string HTTP_X_FORWARDED_FOR that'd be good